Packs/PrismaCloud/IncidentTypes/incidenttype-GCP_Kubernetes_Engine_Misconfiguration.json

```diff
@@ -0,0 +1,22 @@
+{
+    "autorun": false,
+    "color": "#2979FF",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "disabled": false,
+    "hours": 0,
+    "hoursR": 0,
+    "id": "GCP Kubernetes Engine Misconfiguration",
+    "locked": false,
+    "name": "GCP Kubernetes Engine Misconfiguration",
+    "playbookId": "Prisma Cloud Remediation - GCP Kubernetes Engine Misconfiguration",
+    "readonly": false,
+    "reputationCalc": 0,
+    "system": false,
+    "version": -1,
+    "fromVersion": "5.0.0",
```

Comment:
fromversion value should be the same in all pack's entities (in classifier-Prisma_Cloud.json fromversion = 6.0.0 )

Reply:
fixed.